Cubical/Categories/Limits/BinProduct/More.agda

```diff
@@ -205,6 +240,12 @@ module _ (C : Category ℓ ℓ') where
       ×pF-with-agrees : ×Bif ⟪ C .id , f ⟫× ≡ PWN.×pF ⟪ f ⟫
       ×pF-with-agrees = sym (×Bif .Bif-R×-agree _)
 
+  module BinProducts'Notation (bp : BinProducts') =
+    Notation (BinProducts'ToBinProducts bp)
+
+-- TODO: define Notation in terms of BinProduct c c',
```

Comment:
I'm thinking about doing this

Reply:
did you?

Reply:
yes, remarked at https://github.com/maxsnew/cubical-categorical-logic/pull/94#discussion_r1651574537